src/components/views/rooms/RoomTile2.tsx

```diff
@@ -157,7 +160,9 @@ export default class RoomTile2 extends React.Component<IProps, IState> {
         this.setState({notificationsMenuPosition: target.getBoundingClientRect()});
     };
 
-    private onCloseNotificationsMenu = () => {
+    private onCloseNotificationsMenu = (ev: InputEvent) => {
+        ev.preventDefault();
+        ev.stopPropagation();
```

Comment:
onFinished doesn't always have an event, so this causes an exception and also breaks consecutive context menus on a context menu: https://github.com/matrix-org/matrix-react-sdk/blob/769cf1c9870b3ebcb75e5a96e30fbc141efb1792/src/components/structures/ContextMenu.js#L203

Reply:
Think best approach would be to prevent default and stop propagation on this handler: https://github.com/matrix-org/matrix-react-sdk/blob/769cf1c9870b3ebcb75e5a96e30fbc141efb1792/src/components/structures/ContextMenu.js#L322

Reply:
the one that already has it? From testing we just need an `if` statement.